docs/user/aws/install.md

```diff
@@ -60,5 +63,110 @@ The OpenShift console is available via the kubeadmin login provided by the insta
+VPC configuration and a CloudFormation template is provided [here](../../../upi/aws/cloudformation/01_vpc.yaml). 
+
+A created VPC via the template or manually should approximate a setup similar to this:
+
+![Example VPC Architecture Diagram](images/install_upi_vpc.png)
+
+### Create DNS entries and Load Balancers for Control Plane Components
+
+The recommended DNS and load balancer configuration within a CloudFormation template is provided 
+[here](../../../upi/aws/cloudformation/02_cluster_infra.yaml). It uses a public hosted zone and creates a private hosted
+zone similar to the IPI installation method. It also creates load balancers and listeners the same way as the IPI
+installation method. This template can be run multiple times within a single VPC and in combination with the VPC
+template provided above.
+
+#### Create Load Balancer Configuration
+
+It is recommended to create a TCP load balancer for ports 6443 and 22623. The targets will be the master nodes.
+Port 6443 must be accessible to both clients external to the cluster and nodes within the cluster. Port 22623 must
+be accessible to nodes within the cluster. 
+
+#### Create Route53 Hosted Zones & Records
+
+For the cluster name identified earlier in "Create Ignition Configs", you must create a DNS entry which resolves to 
+your created load balancer. The entry api.$clustername.$domain should point to the load balancer.
+
+### Create Security Groups and IAM Roles
+
+The recommended security group and IAM configuration within a CloudFormation template is provided 
+[here](../../../upi/aws/cloudformation/03_cluster_security.yaml). Run this template to get the minimal and permanent
+set of security groups and IAM roles needed for an operational cluster.
+
+TODO: Identify the manual creation requirement/steps.
+
+### Launch Temporary Bootstrap Resource
+
+The recommended bootstrap launch and other necessary, temporary security group plus IAM configuration and a CloudFormation 
+template is provided [here](../../../upi/aws/cloudformation/04_cluster_bootstrap.yaml). Upload your generated bootstrap.ign
+file to an S3 bucket in your account and run this template to get a bootstrap node along with a predictable clean up of 
+the resources when complete. 
+
+TODO: Identify the manual creation requirement/steps.
+
+### Launch Permanent Master Nodes
+
+The recommended master launch and other necessary DNS entries for etcd are provided within a CloudFormation 
+template [here](../../../upi/aws/cloudformation/05_cluster_master_nodes.yaml). Run this template to get a three master
+nodes. 
+
+TODO: Identify the manual creation requirements/steps.
+
+### Monitor for bootstrap-complete and initialization
+
+TODO: Identify the UPI command and expected output in openshift-install
+
+### Cleanup Machine API Resources
+
+TODO: Identify deleting the pre-populated, disconnected machine entries.
```

Comment:
Having this as a post-Install cleanup step is silly.  We should just not generate the resource for this install sequence, and let users decide what machines they want the machine api to adopt or mint thereafter.

Reply:
> Having this as a post-Install cleanup step is silly...

I'm having trouble parsing this.  What are you saying needs cleaning up?

You'll need to remove the bootstrap resources at this point (after receiving the `bootstrap-complete` event), but that's not silly, that's just "we're no longer bootstrapping".  For machine-API adoption, that's currently the `clusterid` tag (#479, although I still wish the tag had better namespacing like `openshift.io/cluster/{name}: owned` or we just used `kubernetes.io/cluster/{name}: owned` for adoption).  If you don't want machines adopted, just don't set that tag.